wizeline/project-lab
# Fix/change count#201

#### What does this PR do?
Change votes count from incremental field to relational count



#### Where should the reviewer start?
- app/pages/projects/[projectId]/index.tsx
- app/projects/mutations/upvoteProject.ts
- app/projects/queries/searchProjects.ts
- db/migrations/20220218180709_removed_votes_count_field/migration.sql
- db/schema.prisma



#### How should this be manually tested?

- Start the application
- Open any project
- Like or dislike the project to check if the behavior is correct



#### Any background context you want to provide?



The schema of the Projects model has been modified in order to remove the field votesCount, this has been done to avoid inconsistencies on the logic of like an dislike button. All the changes are only on the backend logic and shouldn't affect the user interaction.

#### What are the relevant tickets?

[Change votes count from incremental field to relational count #201](https://github.com/wizeline/project-lab/issues/201)



#### Screenshots
NA


#### Questions



#### Checklist
